packages/hackathon/src/data/add_user/sagas.ts

```diff
@@ -50,7 +50,7 @@ const HACKATHON_USER_ATTR_NAME = 'hackathon' // case sensitive
 const HACKATHON_USER_ATTR_LABEL = 'Hackathon'
 const HACKATHON_GROUP_PREFIX = 'Looker_Hack: '
 const LANDING_PAGE_ATTR_NAME = 'landing_page'
-const LANDING_PAGE_PATH = '/extensions/hackathon::hackathon_app'
+const LANDING_PAGE_PATH = '/extensions/hackathon::hackathon'
```

Comment:
Is the user registration script setting this path for the default landing page? Are all existing registrants going to have their homepage updated to this new URL?

Reply:
The user registration group creates a group for hackathon 2021. The script assigns a role  to the group that defines the landing page. When users are added, they are added to the group and thus get the landing page attribute overwritten by the group role. 
So technically, this change shouldn't be needed. And i can jsut change our manifest.kml to rename to `hackathon_app`. But i want to be safe.